src/test/components/__snapshots__/Root.test.js.snap

```diff
@@ -1,231 +1,258 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`app/ProfileViewWhenReady renders an home when the user pressed back after an error 1`] = `
```

Comment:
These changes are because of the `<Fragment>`, but are really mostly whitespace changes as you can see if you change the diff settings.